Python/README.md

```diff
@@ -9,6 +9,101 @@
 
 # Codes in the Python language
 
+## Data Structures
```

Comment:
Even your forked branch with the master and revert all the changes made to the README.md file. Just add a link of your file. Do not make changes to any of the existing links.

Reply:
@atarax665 Done.